### PR TITLE
Measured parameters for DSWx-HLS

### PR DESCRIPTION
**Pull request by JimHofman**
## Description
- Added code for the Measured parameters for DSWx-HLS

## Affected Issues
- N/A
## Testing
- All pytest completed successfully

### CODE REVIEW

**Review by RKuttruff — CHANGES_REQUESTED** [review]
Code changes look good!
Could you please commit `pge/dswx_hls/templates/dswx_hls_measured_parameters.yaml` before merging?

**Review by RKuttruff — APPROVED** [review]

File: src/opera/pge/dswx_hls/templates/dswx_hls_measured_parameters.yaml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  attribute_data_type: int
+  attribute_type: qualityInformation
+  description: The percentage of the tile area with observational data (as opposed to fill value and ocean masked)
+  display_name: SpatialCoverage
+SPATIAL_COVERAGE_EXCLUDING_MASKED_OCEAN:
+  attribute_data_type: int
+  attribute_type: qualityInformation
+  description: The percentage of the tile area with observational data (as opposed to fill value and ocean masked) excluding masked ocean
+  display_name: SpatialCoverageExcludingMaskedOcean
+WORLDCOVER_COVERAGE:
```
Comment (collinss-jpl) [inline src/opera/pge/dswx_hls/templates/dswx_hls_measured_parameters.yaml:186]: Missing entry for WORLDCOVER_SOURCE

**Review by collinss-jpl — CHANGES_REQUESTED** [review]
While inspecting the ISO XML product created by this branch, I noticed several `MeasuredParameters` entries that had `!Not Found!` for one or more fields. I have tried to indicate what the missing entries are in my comments below.

**Comment by RKuttruff** [comment]
The `!Not found!` check passed unit tests because I accidentally used `!Not Found!` in the augment function (capital F not lowercase)
Will refactor to use a constant instead of literal strings at all checks

File: src/opera/pge/dswx_hls/templates/dswx_hls_measured_parameters.yaml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  display_name: AerosolNotWaterToHighConfWaterFMaskValues
+AEROSOL_PARTIAL_SURFACE_AGGRESSIVE_TO_HIGH_CONF_WATER_FMASK_VALUES:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: HLS Fmask values to convert partial surface water aggressive to high-confidence water in the presence of high aerosol
+  display_name: AerosolPartialSurfaceAggressiveToHighConfWaterFMaskValues
+AEROSOL_PARTIAL_SURFACE_WATER_CONSERVATIVE_TO_HIGH_CONF_WATER_FMASK_VALUES:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: HLS Fmask values to convert moderate-confidence water to high-confidence water in the presence of high aerosol
```
[the remark this answers is not in the document]
Comment (RKuttruff) [reply src/opera/pge/dswx_hls/templates/dswx_hls_measured_parameters.yaml:24]: @collinss-jpl 
Turns out the `AEROSOL_PARTIAL_SURFACE_WATER_CONSERVATIVE_TO_HIGH_CONF_WATER_FMASK_VALUES` entry wasn't missing, it had the wrong display name. Perhaps a copy-paste error: it was displayed as a duplicate `AerosolWaterModerateConfToHighConfWaterFMaskValues` attribute.

[the remark this answers is not in the document]
Comment (RKuttruff) [reply src/opera/pge/dswx_hls/templates/dswx_hls_measured_parameters.yaml:24]: It looks like this existed with the old version of the template:
image

**Review by collinss-jpl — APPROVED** [review]
Looks good @RKuttruff, nice work diagnosing the issue with the missing/duplicate parameter in the original template.

**Pull request merged**